src/app/components/error/conflict/conflict.component.html

```diff
@@ -1,5 +1,7 @@
 <app-govuk-heading> {{ header || 'This request is no longer available' }}</app-govuk-heading>
-<p id="conflict-body" class="govuk-body">{{ body || 'Another user may have already actioned this request.' }}</p>
+<p id="conflict-body" class="govuk-body" tabindex="0">
```

Comment:
Wouldn't this tabindex need to be 1 as the heading is 0?

Reply:
tabindex=0 inserts the element into the natural ordering of the webpage, meaning it will sit based on where it is in the template, tabindex=1 can have some pretty bizarre side-effects